facia-purger/cloudformation/cloudformation.json

```diff
@@ -115,7 +115,65 @@
           ]
         },
         "Runtime": "java8",
-        "Timeout": 20
+        "Timeout": 30
+      }
+    },
+    "NotificationTopic": {
+      "Type": "AWS::SNS::Topic",
+      "Properties": {
+        "Subscription": [ {
+          "Protocol": "email",
+          "Endpoint": "dotcom.platform@guardian.co.uk"
```

Comment:
Can we make this a parameter?
Is it even possible to make this param a comma separated list in case we want to add more recipients?